dev-haskell/ghc-check/ghc-check-0.5.0.3.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
```

Comment:
You can add a line here like `#hackport: flags: ghc-check-use-package-abis:package-abis` to maintain the flag rename between version bumps.